flow/templates/expanse.sh

```diff
@@ -0,0 +1,54 @@
+{# Templated in accordance with: https://www.sdsc.edu/support/user_guides/expanse.html #}
+{% extends "slurm.sh" %}
+{% block tasks %}
+    {% set threshold = 0 if force else 0.9 %}
+    {% set cpu_tasks = operations|calc_tasks('np', parallel, force) %}
+    {% set gpu_tasks = operations|calc_tasks('ngpu', parallel, force) %}
+    {% if gpu_tasks %}
+        {% if not ('gpu' in partition or force) %}
+            {% raise "GPU operations require a GPU partition!" %}
+        {% endif %} {# End check for correct partition #}
+        {# GPU nodes have 4 NVIDIA V100-32GB SMX2 #}
+        {% set nn_gpu = gpu_tasks|calc_num_nodes(4) %}
+        {% set nn = nn_gpu %}
+    {% else %} {# if no GPU tasks #}
```

Comment:
Personally I find "endif" comments uninformative and a bit confusing when the template already has indented logic. It only begins the `if` block 7 lines above, which is much less than one screen-full, so it's not like the original condition is out of sight.
```suggestion
    {% else %}
```